backend/src/main/java/com/redhat/cloud/notifications/db/ApplicationResources.java

```diff
@@ -105,11 +105,10 @@
         });
     }
 
-    // Note: This method uses a stateless session
     public Uni<EventType> getEventType(String bundleName, String applicationName, String eventTypeName) {
         final String query = "FROM EventType WHERE name = :eventTypeName AND application.name = :applicationName AND application.bundle.name = :bundleName";
-        return sessionFactory.withStatelessSession(statelessSession -> {
-            return statelessSession.createQuery(query, EventType.class)
+        return sessionFactory.withSession(session -> {
```

Comment:
This was previously used by a REST endpoint and during Kafka messages processing. Now, it is only used by the REST endpoint so we don't need the stateless session anymore.